src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class RightTriangle extends FigCol {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(double firstLeg, double secondLeg, Color color) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    public double getFirstLeg(double firstLeg) {
```

Comment:
The getter method 'getFirstLeg' should not take any parameters. Getters are intended to return the current value of a class's field.